### PR TITLE
feat: form elements - textarea documentation

### PR DESCRIPTION
**Pull request by catiarodriguescosta**
- Moved component to a new page
- Added properties and examples for wmndsTextarea component
- Updated all references of textarea
- Updated textarea params
- Grouped params: error message params, label params, group params and attributes params (like Input)
- Update error message and label locally
- Error example

### CODE REVIEW

File: src/wmnds/components/textarea/_textarea.njk [hunk 1]
```diff
@@ -0,0 +1,64 @@
+{% macro wmndsTextarea(params) %}
```
Comment (houbly) [inline src/wmnds/components/textarea/_textarea.njk]: Could you put this folder back into the form-elements for now?

File: src/wmnds/components/textarea/_textarea.njk [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+{# Params #}
+{% set id = params.id if params.id else "textarea" %}
+{% set name = params.name if params.name else id %}
+{% set rows = params.rows if params.rows else 3 %}
+{% set required = params.required if params.required else false %}
+{% set classes = " " + params.classes if params.classes %}
+{# Error Message Params #}
+{% set groupErrorClass = " wmnds-fe-group--error" if params.error %}
+{% set textareaErrorClass = " wmnds-fe-input--error" if params.error %}
+{% set errorContentText = params.errorMessage.contentText if params.errorMessage.contentText else null %}
```
Comment (houbly) [inline src/wmnds/components/textarea/_textarea.njk]:
```suggestion
{% set errorContentText = params.errorMessage.contentText if params.errorMessage.contentText else "Please enter some text" %}
```

File: src/wmnds/components/textarea/_properties.njk.json [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+      "arrayOptions": [
+        {
+          "name": "classes",
+          "type": "string",
+          "description": "Classes to add to the form group (e.g. to show error state for the whole group)"
+        }
+      ]
+    },
+    {
+      "name": "Attributes",
```
Comment (houbly) [inline src/wmnds/components/textarea/_properties.njk.json]:
```suggestion
      "name": "attributes",
```
lowercase a

**Review by houbly — CHANGES_REQUESTED** [review]

File: src/wmnds/components/textarea/_textarea.njk (hunk 1 above)
Comment (catiarodriguescosta) [reply src/wmnds/components/textarea/_textarea.njk]: Yeah, just did it a few minutes ago! 👍  We are synchronized!

File: src/wmnds/components/textarea/_textarea.njk (hunk 2 above)
Comment (catiarodriguescosta) [reply src/wmnds/components/textarea/_textarea.njk]: I'll commit it but it has a fallback inside the errorMessage component. 👍  but that way is a bit more customized for textarea. :)

File: src/wmnds/components/textarea/_properties.njk.json (hunk 3 above)
Comment (catiarodriguescosta) [reply src/wmnds/components/textarea/_properties.njk.json]: Change done in the last commit. As I changed the folder location suggestion wasn't available anymore.

**Review by houbly — APPROVED** [review]

**Pull request merged**